src/sidebar/search-client.js

```diff
@@ -47,6 +47,7 @@ export default class SearchClient extends TinyEmitter {
     this._canceled = false;
     /** @type {Annotation[]} */
     this._results = [];
+    this._resultCount = 0;
```

Comment:
Since `0` is a valid number of results that we might get back, I'd be inclined to use `null` instead to represent the "we don't know the number of results" state.

Reply:
Yeah, I agree. Will change.